docs/powerpoint/work-with-tables.md

```diff
@@ -0,0 +1,434 @@
+        fill: { color: "darkslateblue" },
+        font: { bold: true, color: "white" }
+    }
+};
+await insertTableOnCurrentSlide(rowCount, columnCount, options);
+```
+
+The previous sample creates a table as shown in the following image.
+
+:::image type="content" source="../images/powerpoint-table-uniform-cell-format.png" alt-text="A PowerPoint table with a fill color of dark slate blue, and white bold font.":::
+
+### Specific cell formatting
+
+Specific cell formatting applies to individual cells and overrides the uniform cell formatting, if any. Set individual cell formatting by using the `TableAddOptions.specificCellProperties` property. The following code sample shows how to set the fill color to black for the cell at row 1, column 1. About the following code sample, note the following.
+
+The `specificCellProperties` must be a 2D array that matches the 2D size of the table exactly. The sample first creates the entire empty 2D array of objects. Then it sets the specific cell format at row 1, column 1, after the options object is created.
+
+```javascript
+const rowCount = 3;
+const columnCount = 4;
+// Compact syntax to create a 2D array filled with empty and distinct objects.
+const specificCellProperties = Array(rowCount).fill("").map(_ => Array(columnCount).fill("").map(_ => ({})));
+const options: PowerPoint.TableAddOptions = {
+    values: [
+        ["1", "2", "", "4"],
+        ["5", "6", "7", "8"],
+        ["9", "10", "11", "12"]
+    ],
+    uniformCellProperties: {
+        fill: { color: "darkslateblue" },
+        font: { bold: true, color: "white" }
+    },
+    specificCellProperties // Array values are empty objects at this point.
+};
+// Set fill color for specific cell at row 1, column 1.
+options.specificCellProperties[1][1] = {
+    fill: { color: "black" }
+};
+await insertTableOnCurrentSlide(rowCount, columnCount, options);
+
+```
+
+The previous sample creates a table with a specific format applied to the cell in row 1, column 1 as shown in the following image.
+
+:::image type="content" source="../images/powerpoint-table-specific-cell-format.png" alt-text="A PowerPoint table with the cell in row 1 column 1 formatted with a black background.":::
+
+The previous sample uses the `font` property which is of type `TableCellFontProperties`. The `font` property allows you to specify many properties, such as bold, italic, name, color, and more. The following code sample shows how to specify multiple properties for a font for a cell.
+
+```javascript
+options.specificCellProperties[1][1] = {
+    font: {
+        color: "orange",
+        name: "Arial",
+        size: 50,
+        allCaps: true,
+        italic: true,
+    }
+};
+```
+
+You can also specify a `fill` property which is of type `TableCellFillProperties`. The `fill` property can specify a color and the transparency percentage. The following code sample shows how to create a fill for all table cells using the color "light red" and a 50% transparency.
```

Comment:
Same here - maybe TableCellFillProperties is an internal type - this isn't available to the public. What we have in the docs is that TableCellProperties.fill is of type FillProperties.

```suggestion
You can also specify a `fill` property which is of type `FillProperties`. The `fill` property can specify a color and the transparency percentage. The following code sample shows how to create a fill for all table cells using the color "light red" and a 50% transparency.
```